config/initializers/apipie.rb

```diff
@@ -0,0 +1,8 @@
+Apipie.configure do |config|
+  config.app_name                = "Falko20172"
+  config.api_base_url            = ""
+  config.doc_base_url            = "/apipie"
+  config.translate               = false
+  # where is your API defined?
```

Comment:
Is this really needed?

Reply:
This is required to configure apipie and define a url to show all endpoints.

Reply:
I mean the comment `# where is your API defined?`.

Reply:
Really wasn't necessary, I removed the comment.